src/main/java/org/broadinstitute/consent/http/mail/freemarker/FreeMarkerTemplateHelper.java

```diff
@@ -80,6 +80,17 @@ public Writer getDataCustodianApprovalTemplate(List<DatasetMailDTO> datasets, St
         return generateDataCustodianApprovalTemplate(datasets, dataDepositorName, darCode, researcherEmail, temp);
     }
 
+    public Writer getDatasetApprovedTemplate(String dataSubmitterName, String datasetName, String dacName) throws IOException, TemplateException {
+        Template temp = freeMarkerConfig.getTemplate("dataset-approved.html");
```

Comment:
More thinking out loud ... the html file names should be consolidated somewhere into a constants file/hash/whatever. Not for this PR, but for a refactoring PR.